app/controllers/orders/pending.js

```diff
@@ -13,10 +13,23 @@ export default Controller.extend({
     return this.get('model.order.paymentMode') === 'paypal';
   }),
 
+  isOmise: computed('model.order', function() {
+    return this.get('model.order.paymentMode') === 'omise';
+  }),
+
   paymentAmount: computed('model.order', function() {
     return this.get('model.order.amount') * 100;
   }),
 
+  publicKeyOmise: computed('settings.omiseLivePublic', function() {
+    return this.get('settings.omiseLivePublic') || this.get('settings.omiseTestPublic');
+  }),
+
+  omiseFormAction: computed('model.order', function() {
+    let identifier = this.get('model.order.identifier');
+    return `http://127.0.0.1:5000/v1/orders/'${identifier}'/omise-checkout`;
```

Comment:
`http://127.0.0.1:5000/v1/orders/'${identifier}'/omise-checkout` replace this with `${hostname}/${identifier}/omise-checkout` . You can find hostname from `environment.js`

Reply:
@mrsaicharan1 Sure! pushing the changes asap